src/styles.scss

```diff
@@ -98,10 +98,12 @@ $service-manual-container-size: 1100px; // Container width variable.
 
 .width-auto { width: auto; }
 .width-15 { width: 15%; }
-.width-25 { width: 25%; }
+.width-25 { width: 25%; } 
+.width-20 { width: 20%; } 
```

Comment:
We should not abuse of these spacings. They exists for very exceptional layouts. not to be used everytime. Be caution...